src/ethereum/cancun/fork.py

```diff
@@ -310,19 +310,8 @@ def validate_header(chain: BlockChain, header: Header) -> None:
     """
     if header.number < Uint(1):
         raise InvalidBlock
-    parent_header_number = header.number - Uint(1)
-    first_block_number = chain.blocks[0].header.number
-    last_block_number = chain.blocks[-1].header.number
-
-    if (
-        parent_header_number < first_block_number
-        or parent_header_number > last_block_number
-    ):
-        raise InvalidBlock
-
-    parent_header = chain.blocks[
-        parent_header_number - first_block_number
-    ].header
+    # Post-merge forks do not allow ommers, so parent_header is always the last header in the chain
+    parent_header = chain.blocks[-1].header
 
```

Comment:
You also need to implement the update to `Paris` and `Prague` which are also post-merge